.github/workflows/build.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           pip install pylint-flask-sqlalchemy
       - name: Run python mypy
         run:
-          python mypy.py
+          python init_test_data.py
```

Comment:
This is not so good way to initialize database with test data. You can use migration or fixtures